.semgrepignore

```diff
@@ -7,6 +8,12 @@
 **/*.min.js
 **/env.configs.yml
 
+# ignore template files
```

Comment:
these template files had warnings for some of the controllers not having authorization but i figure for templates it doesnt matter. same thing with some of the ones in test